drivers/adc/ad7779/ad7779.c

```diff
@@ -319,7 +320,33 @@ int32_t ad7779_set_spi_op_mode(ad7779_dev *dev,
 int32_t ad7779_get_spi_op_mode(ad7779_dev *dev,
 			       ad7779_spi_op_mode *mode)
 {
-	*mode = dev->spi_op_mode;
+	int ret;
```

Comment:
Use the same type as the one returned by the function (int32_t).

Reply:
changed `int` ret to` int32_t ret` wherever relevant